card/gem_mods/ace_editor/card-mod-ace_editor.gemspec

```diff
@@ -0,0 +1,21 @@
+# -*- encoding : utf-8 -*-
+
+Gem::Specification.new do |s|
+  s.name = "card-mod-ace_editor"
+  s.version = "0.1"
+
+  s.authors = ["Ethan McCutchen", "Philipp Kühl"]
```

Comment:
Please make yourself third author on these.

Reply:
I think i missed one comment for completion, but this is done.